projects/lf-documentation/src/index.html

```diff
@@ -6,6 +6,7 @@
   <title>LfDocumentation</title>
   <meta name="viewport" content="width=device-width, initial-scale=1" />
   <link rel="icon" type="image/x-icon" href="favicon.ico" />
+  <link href="https://cdn.jsdelivr.net/npm/@laserfiche/lf-ui-components@14/cdn/indigo-pink.css" rel="stylesheet"/>
```

Comment:
I gave you the wrong url. This is the external CDN. We should be using our Laserfiche CDN for security purposes
Update to `https://lfxstatic.com/npm/@laserfiche/lf-ui-components@14/cdn/indigo-pink.css`